libs/set-algebra/CHANGELOG.md

```diff
@@ -14,11 +14,11 @@
 
 ## 1.1.0.1
 
-- Set upper bound ansi-wl-pprint < 1.0
+* Set upper bound ansi-wl-pprint \< 1.0
 
 ## 1.1.0.0
 
-- Remove unused instances for `Data.UMap` #3371
+* Remove unused instances for `Data.UMap` #3371
```

Comment:
In almost all the files occurrences of `*` has been replaced with `-`. But in three files alone it is the opposite change.

Reply:
I think it is the nested bullet points that have been change.
So, @neilmayhew, correct me if I am wrong, but my understanding is that the top level uses `*`, while nested bullet points use `-`, right?

I am totally fine with this approach as long as we are consistent about it and there is a linter to enforce this :smiley:

Reply:
I don't think it makes sense to have deeply nested bullet points in changelogs